blog/zh/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md

```diff
@@ -100,6 +105,21 @@ Tau T2A VM 有多种预定义的 VM 形状，每个 VM 最多 48 个 vCPU，每
 | AWS C7g          | $0.0361 | $0.0723 | $0.1445 | $0.289 | $0.5781 | $1.1562 | $1.7342 |
 | GCP T2A          | $0.0385 | $0.077  | $0.154  | $0.308 | $0.616  | $1.232  | $1.848  |
 | Azure Dpsv5      | *       | $0.077  | $0.154  | $0.308 | $0.616  | $1.232  | $1.848  |
+| OCI A1           | 0       |0        |0        | $0.051 | $0.179  | $0.435  | $0.947  |
+
+由于 OCI VM.Standard.A1.Flex 可以灵活配置内存和 CPU 核心数，因此下面计算性价比时选取的机器内存大小为：
```

Comment:
<img width="1133" alt="image" src="https://user-images.githubusercontent.com/39793568/196368446-f27314ec-83aa-4fd9-badf-64b219d84fec.png">
I think this part is either a separate text, or it is directly highlighted in the `note`, instead of the current sentence divided into two formats